recipes/anyrpc/all/test_package/test_anyrpc.cpp

```diff
@@ -0,0 +1,12 @@
+#include <cstdlib>
+#include <iostream>
+
+#include "anyrpc/anyrpc.h"
+
+int main(void)
+{
+    anyrpc::JsonHttpServer server;
+    std::cout << "Success!" << std::endl;
```

Comment:
```suggestion
    anyrpc::InitializeLogger();
    anyrpc::log_time(INFO);
```
Try to exercise some method to validate runtime linkage.

Reply:
InitializeLogger and log_time are macro and only call log4cplus if BUILD_WITH_LOG4CPLUS is defined, otherwise they do nothing.
But ok I added some method calls.